src/main/resources/uk/gov/hmcts/befta/dse/ccd/definitions/valid/BEFTA_MASTER/FT_NoCAutoApprovalCaseType/CaseField.json

```diff
@@ -114,7 +114,7 @@
   "LiveFrom" : "01/01/2017",
   "LiveTo" : "",
   "CaseTypeID" : "FT_NoCAutoApprovalCaseType",
-  "ID" : "changeOrganisationRequestField",
+  "ID" : "ChangeOrganisationRequest",
```

Comment:
Renaming the COR fields for `FT_NoCAutoApprovalCaseType`, `FT_NoCCaseType` & `FT_NoCWithoutEventsCaseType` will impact a lot of FTA tests across the NoC APIs.  **So is this rename strictly necessary** for the [RDM-10262](https://tools.hmcts.net/jira/browse/RDM-10262) _"Update ChangeOrganisationRequest import validations"_?  

NB: The new COR validation should be on `FieldType` not field name. see [Request NoC LLD](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1452902365#APIOperation:RequestNoticeofChange-2.5-Low-LevelFlow) which shows a specific step to lookup the COR field name. 
> Note: The CCD Definition Store is configured with global complex field types to support the NoC processes, i.e. OrganisationPolicy and ChangeOrganisationRequest; however each CCS Services have the freedom to define the field names they will use to hold this data. Therefore as the field types are not available to this API at this stage of the process, the ChangeOrganisationRequest property can be identified by calling the Start Event API (EI-5) which gives us a break down of all the case fields by type and we lookup the field ID based on the type

Reply:
After speaking with the BA for this ticket, this validation requirement will be removed